pkg/resources/machinecontroller/deployment.go

```diff
@@ -263,6 +263,8 @@ func getFlags(clusterDNSIP string, nodeSettings kubermaticv1.NodeSettings, exter
 	if nodeSettings.PauseImage != "" {
 		flags = append(flags, "-node-pause-image", nodeSettings.PauseImage)
 	}
+
+	// TODO(kron4eg): deprecate and remove this
 	if nodeSettings.HyperkubeImage != "" {
 		flags = append(flags, "-node-hyperkube-image", nodeSettings.HyperkubeImage)
 	}
```

Comment:
I believe that we might want to add the ability to configure the Kubelet image instead of this.

Reply:
Yeah, but first we need to upgrade machine-controller, so that's another task to do.